server/database/build.sql

```diff
@@ -0,0 +1,42 @@
+BEGIN;
+
+DROP TABLE IF EXISTS voters,users,box,region;
```

Comment:
We are still making changes in the DB (voters name may be one field instead of three) that is why we kept this line for now. At the production version this line would not appear.

Reply:
The point is you don't want to wipe your database clean every time

Reply:
Done :+1: